pdc/apps/package/serializers.py

```diff
@@ -46,6 +79,24 @@ def to_internal_value(self, data):
             data['filename'] = models.RPM.default_filename(data)
         return super(RPMSerializer, self).to_internal_value(data)
 
+    def create(self, validated_data):
+        dependencies = validated_data.pop('dependencies', [])
```

Comment:
I don't think so. The `DependencySerializer` field takes dict on input, but after validating it returns just a list, as the split into 6 different lists is no longer needed. I guess this should be better documented for developers.